aws/sdk/examples/ec2/src/bin/list-all-instance-events.rs

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use aws_config::meta::region::RegionProviderChain;
+use aws_sdk_ec2::{Client, Error, Region, PKG_VERSION};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// The AWS Region.
+    #[structopt(short, long)]
+    region: Option<String>,
+
+    /// Whether to display additional runtime information.
+    #[structopt(short, long)]
+    verbose: bool,
+}
+
+/// Shows the scheduled events for the Amazon Elastic Compute Cloud (Amazon EC2) instances in the Region.
+async fn show_events(reg: &'static str) {
+    /*
+        let region = Region::new(reg.clone());
+        let config = aws_sdk_ec2::Config::builder().region(region).build();
+        let client = Client::from_conf(config);
+    */
+
+    //let reg_slice: &str = Box::leak(reg.into_boxed_str());
+
```

Comment:
Did you intend to keep these commented lines?
```suggestion
```

Reply:
Removed.